vic/drivers/shared_image/src/vic_restore.c

```diff
@@ -882,6 +883,9 @@ vic_restore(void)
         }
     }
 
+    // routing ring
+    vic_restore_extension(&(filenames.init_state), state_metadata);
```

Comment:
rename function to `vic_restore_rout_extension`

Reply:
done. also changed `vic_store_extension` into `vic_store_rout_extension`